site/_posts/2022-09-09-what-is-webauthn-why-do-you-care.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: blog-post
+title: What is WebAuthn and why do you care?
+description: WebAuthn is a new standard which secures user accounts and improves user experience. What could be better?
+author: Dan Moore
+image: blogs/sso-explanation/how-sso-works.png 
```

Comment:
Do we have another image for this?

Reply:
Will have a correct header image before shipping.